src/main/proto/netflix/titus/control_plane_agent.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+// Data entities exchanged between Titus control plane and its agents.
+//
+
+/// Information about network interface associated with a container.
+message NetworkConfiguration {
+
+    /// Assigned IP address.
+    string ipAddress = 1;
+
+    /// ENI id.
+    string eniId = 2;
```

Comment:
Is this the "eni-abcd..."?

Reply:
Yes, it is AWS ENI id. We store it today in the task context.

Reply:
So, is the "resource-eni-0" thing not going to be sent back anymore?

Reply:
It is ENI index assigned by the Titus scheduler with `resource-eni-` prefix added to it. We parse this value today, and set as `task.networkInterfaceIndex` in a task context. We have this information already in `TwoLevelResource` object, so this is redundant information.